mithril-explorer/src/components/Artifacts/CardanoDbSnapshotsList/index.js

```diff
@@ -53,14 +53,15 @@ export default function SnapshotsList(props) {
 
       <div className={props.className}>
         <h2>
-          Snapshots <RawJsonButton href={artifactsEndpoint} variant="outline-light" size="sm" />
+          Cardano Db Snapshots{" "}
```

Comment:
Not directly related to the PR, but are these spaces really necessary in the artifacts components?

Reply:
Well in this case the icon border is really visible since we use a [`outline-light`](https://react-bootstrap.github.io/docs/components/buttons#outline-buttons) variant, so for the sake of the demonstration I changed it to `outline-secondary`:

- Without the space:
![image](https://github.com/input-output-hk/mithril/assets/790521/05af1d05-3a1c-4fd8-8fff-c5552b613122)

- With the space:
![image](https://github.com/input-output-hk/mithril/assets/790521/2fc03038-e181-4644-9413-d80a225aa02e)